lib/csp.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+var thing = require('core-util-is');
+
```

Comment:
naming is hard, but shouldn't this be a little better than "thing"? like "cspPolicyType"?